src/pages/Admin/Venue/Users/VenueUsersReport.tsx

```diff
@@ -0,0 +1,166 @@
+          {worldUsers.map((user: any, index) => {
+            return (
+              userIsComplete(user) && (
+                <tr key={index}>
+                  <td>{index + 1}</td>
+                  <td>{user.partyName || "Not"}</td>
+                  <td>{user.realName || "Not set"}</td>
+                  <td>{user.companyDepartment || "Not set"}</td>
+                </tr>
+              )
+            );
+          })}
+        </tbody>
+      </Table>
+    );
+  };
+
+  const downloadCSV = useCallback(() => {
+    downloadGeneratedCSVFile(worldUsers, headersSelected);
+  }, [worldUsers, headersSelected]);
+
+  const renderDownloadBtn = () => {
+    return (
+      <button
+        className="btn btn-primary download-btn"
+        onClick={() => downloadCSV()}
+      >
+        Download{" "}
+      </button>
+    );
+  };
+
+  const updateSelectedHeaders = useCallback(
+    (selected: string) => {
+      if (headersSelected.includes(selected)) {
+        headersSelected.splice(headersSelected.indexOf(selected), 1);
+        setHeadersSelected([...headersSelected]);
+      } else {
+        setHeadersSelected([...headersSelected, selected]);
+      }
+    },
+    [headersSelected]
+  );
+
+  const addCustomField = useCallback(() => {
+    if (!headers.includes(customFieldValue)) {
+      setHeaders([...headers, customFieldValue]);
+    }
+  }, [customFieldValue, headers]);
+
+  const updateCustomFieldValue = useCallback(
+    (field: string) => {
+      setCustomFieldValue(field);
+    },
+    [setCustomFieldValue]
+  );
+
+  const renderFoundedHeaders = () => {
+    return (
+      <div className="venue-users-detail-filter-box-headers">
+        {headers.map((header) => (
+          <div
+            className="venue-users-detail-filter-box-headers-item"
+            key={header}
+          >
+            <Form.Check
+              type="switch"
+              id={header}
+              label={header}
+              checked={headersSelected.includes(header)}
+              onChange={() => updateSelectedHeaders(header)}
+            />
+          </div>
+        ))}
+      </div>
+    );
+  };
+
+  const renderCustomHeaders = () => {
+    return (
+      <div className="venue-users-detail-filter-box-custom-field">
+        <label>Add custom field:</label>
+        <input
+          onChange={(event) => updateCustomFieldValue(event.target.value)}
+        />
+        <button className="btn btn-primary" onClick={() => addCustomField()}>
+          Add field
+        </button>
+      </div>
+    );
+  };
+
+  const renderFieldFilter = () => {
+    return (
+      <div className=" venue-users-detail-filter-box">
+        {renderFoundedHeaders()}
+        {renderCustomHeaders()}
+        {renderDownloadBtn()}
+      </div>
+    );
+  };
+
+  return (
+    <div className="venue-users-detail">
+      {renderPageHeader()}
+      {renderFieldFilter()}
+      {renderUserTable()}
+    </div>
+  );
+};
+
+export default VenueUsersReport;
```

Comment:
We use explicitly named exports, never `default`.

```suggestion
};
```

Reply:
fixed in 12e5c07fea64a78aced31d706faf59c5e523c733